packages/traceloop-sdk/traceloop/sdk/dataset/model.py

```
@@ -0,0 +1,106 @@
+import datetime
+from enum import Enum
+from pydantic import BaseModel, Field, ConfigDict
```

Ruff (0.12.2)

`pydantic.ConfigDict` imported but unused

Remove unused import: `pydantic.ConfigDict`

(F401)